routes/ai_brain.py

```diff
@@ -0,0 +1,77 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from typing import List
+from .. import models, schemas
+from fastapi.responses import FileResponse
+from ..database import get_db  
+
+router = APIRouter(prefix="/api/knowledge-base", tags=["Knowledge Base"])
+
+
+# LIST ALL ENTRIES
```

Comment:
okay but no api for getting count for each type of data for every user which is a necessary api.